### PR TITLE
Displays arrow-up when filter visibility toggles

### PR DESCRIPTION
**Pull request by ankurshr01**
Added arrow-up, which is shown when the filter section is extended.

Initial :- 

Screenshot (172)

Final :- 

Screenshot (173)

### CODE REVIEW

**Comment by ankurshr01** [comment]
@akmadian  done :)

**Comment by ankurshr01** [comment]
@akmadian  check !

**Comment by ankurshr01** [comment]
@akmadian Sir I have used SVG files, kindly review :)

**Comment by ankurshr01** [comment]
@akmadian Done !

File: README.md [hunk 1]
```diff
@@ -50,7 +50,7 @@ If you want to make your local development server accessible to the internet (fo
 ngrok http http://localhost:8443 -host-header="localhost:8443"
 ```
 
-If you need to run a HTTP version (for example, if you're testing against third-party websites that do not accept the self-signed cerificate generated by the dev server), run the dev server using `npm run dev-insecure` and use the following command to start `ngrok`:
+If you need to run a HTTP version (for example, if you're testing against third-party websites that do not accept the self-signed cerificate generated by the dev server), run the dev server using `npm run dev` and use the following command to start `ngrok`:
```
Comment (brenoferreira) [inline README.md]: Can you please undo these changes? This has already been fixed

File: src/assets/arrow.svg [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="10pt" height="10pt" viewBox="0 0 10 10" version="1.1">
+<g id="surface1">
+<path style=" stroke:none;fill-rule:nonzero;fill:rgb(0%,0%,0%);fill-opacity:1;" d="M 9.566406 2.46875 L 9.246094 2.148438 C 9.148438 2.046875 9.015625 1.992188 8.875 1.992188 C 8.730469 1.992188 8.597656 2.046875 8.496094 2.148438 L 4.863281 5.785156 L 1.226562 2.144531 C 1.125 2.046875 0.992188 1.992188 0.847656 1.992188 C 0.707031 1.992188 0.574219 2.046875 0.476562 2.144531 L 0.15625 2.464844 C -0.0507812 2.671875 -0.0507812 3.007812 0.15625 3.21875 L 4.488281 7.5625 C 4.585938 7.660156 4.71875 7.730469 4.863281 7.730469 C 5.007812 7.730469 5.140625 7.660156 5.242188 7.5625 L 9.566406 3.226562 C 9.667969 3.128906 9.722656 2.988281 9.722656 2.847656 C 9.722656 2.703125 9.667969 2.570312 9.566406 2.46875 Z M 9.566406 2.46875 "/>
```
Comment (brenoferreira) [inline src/assets/arrow.svg]: Not sure why the SVG changed. What's wrong with the old one?

File: src/components/FilterChecklist.vue [hunk 3]
```diff
@@ -2,9 +2,8 @@
   <fieldset>
     <div class="filters-title" @click.prevent="toggleFilterVisibility">
       <span>{{ title }}</span>
-
-      <button class="filter-dropdown-visibility-button">
-        <img src="@/assets/arrow-bottom.svg" class="arrow-bottom" alt="toggle filters visibility" />
+  <button v-bind:class="[filtersVisible ? 'rotImg' : 'filter-dropdown-visibility-button']">
```
Comment (brenoferreira) [inline src/components/FilterChecklist.vue]: It's better to use something like this instead to toggle the `rotImg` class https://vuejs.org/v2/guide/class-and-style.html#Object-Syntax

File: src/components/FilterChecklist.vue [hunk 4]
```diff
@@ -60,12 +58,15 @@ fieldset {
...
 }
-
 .filter-dropdown-visibility-button {
   float: right;
   cursor: pointer;
 }
-
+.rotImg{
+  transform: rotate(180deg);
+   float: right;
```
Comment (brenoferreira) [inline src/components/FilterChecklist.vue]: If you use the class binding I suggested above you can keep this class style with only `transform: rotate(180deg);`

Comment (akmadian) [reply src/components/FilterChecklist.vue]: This, and I think it would also be best to do the class binding on the img element itself instead of the button. (I mentioned this in my last review, along with Breno's request for the README change. Please be sure to read thoroughly and ask questions if you have them!)

Comment (ankurshr01) [reply src/components/FilterChecklist.vue]: Actually I haven't changed the Readme file, is showing because in this PR, there is commit from another user,
So can I create a new PR for this issue? And close this one?

Comment (ankurshr01) [reply src/components/FilterChecklist.vue]: Since I got warning about line length being exceed by 100, so i have used 

@brenoferreira

**Comment by ankurshr01** [comment]
Review @brenoferreira .

**Comment by ankurshr01** [comment]
@brenoferreira 
please review my last update, and suggest if anything left.

File: src/components/FilterChecklist.vue [hunk 5]
```diff
@@ -2,12 +2,11 @@
   <fieldset>
     <div class="filters-title" @click.prevent="toggleFilterVisibility">
       <span>{{ title }}</span>
-
-      <button class="filter-dropdown-visibility-button">
```
Comment (akmadian) [inline src/components/FilterChecklist.vue]: Please fix indentations here. Your button opening tag is a couple levels behind the closing, and the img tag should be one more level in. You can also break the img element down into multiple lines like so: 
```
<img src="@/assets/arrow.svg"
  v-bind:class="{rotImg:filtersVisible}"
  alt="toggle filters visibility"/>
```
This avoids the eslint warning about line length, and avoids disabling all eslint for that line.

Comment (ankurshr01) [reply src/components/FilterChecklist.vue]: @akmadian  Changes done !!
